scripts/publish-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+echo "Travis runs in the $(pwd) directory"
+# Set up the user for commiting and pushing.
+git config --global user.email "<mhippely@vmware.com>"
+git config --global user.name "clr-team"
+
+SHA=$(git rev-parse HEAD)
+MESSAGE=$(git log -1 --pretty=format:%s)
+
+# Clone repositories
+git clone https://${GITHUB_OAUTH_BUILD_TOKEN}@github.com/clr-team/clr-angular.git
```

Comment:
Might as well move this inside of the for loop below. Also add `--depth 1` so it is faster and only clones the latest commit instead of the whole tree.

```
do
  git clone https://${GITHUB_OAUTH_BUILD_TOKEN}@github.com/clr-team/${repo}.git --depth 1
  ...
```

Reply:
I'll move it into the loop and add the depth. I had it in an earlier version and forgot to add it in this one.